### PR TITLE
8.2.5

### PR DESCRIPTION
**Pull request by pookmish**
# What
- Reduce custom code in favor of contrib module support and a drupal core patch
- Added the ability to create a google forms media type to embed google forms iframes.
- Users can now edit captions in the modal within the wysiwyg. This adds to the ability to edit the caption directly in the wysiwyg.

# So what
- Less technical debt and increased functionality

# Now what
- Deploy & Clear cache 
- Create a google forms media type if desired.

# New Requirements and Risks
- None

### CODE REVIEW

**Comment by sherakama** [comment]
oooo. I'll let @imonroe  take this one.

File: config/install/field.field.media.google_form.field_media_google_form.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_google_form
+    - media.type.google_form
+id: media.google_form.field_media_google_form
+field_name: field_media_google_form
+entity_type: media
+bundle: google_form
+label: 'Google Form'
```
Comment (imonroe) [inline config/install/field.field.media.google_form.field_media_google_form.yml:11]: IMO, this should be 'Google Form URL'.  The meaning could be unclear in the UI.

**Comment by pookmish** [comment]
This PR is just the release. this doesn't change or add any functionality

File: config/install/field.field.media.google_form.field_media_google_form.yml (hunk 1 above)
Comment (sherakama) [reply config/install/field.field.media.google_form.field_media_google_form.yml:11]: That is a good question. We should bring that up with Ishita.

**Comment by sherakama** [comment]
@imonroe we have a ticket in the backlog for creating a form paragraph type: https://stanfordits.atlassian.net/browse/D8CORE-1841

I think media with caption can stay along the lines of 'rich media'

**Pull request merged**